tests/strictEqual.test.js

```diff
@@ -0,0 +1,33 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictEqual')
+
+
+describe('strictEqual', () => {
+
+  it('returns false if data types are not equal', () => {
+    const checkOne = 5
+    const checkTwo = '5'
+    const checkResult = strictEqual(checkOne, checkTwo)
+    expect(checkResult).to.equal(false)
+  })
+  it('returns true if strings are equal', () => {
+    const checkOne = 'str'
+    const checkTwo = 'str'
+    const checkResult = strictEqual(checkOne, checkTwo)
+    expect(checkResult).to.equal(true)
+  })
+  it('returns false if strings are not equal', () => {
+    const checkOne = 'str1'
+    const checkTwo = 'str2'
+    const checkResult = strictEqual(checkOne, checkTwo)
+    expect(checkResult).to.equal(false)
+  })
+  it('returns true if numbers are equal', () => {
+    const checkOne = 5
+    const checkTwo = 5
+    const checkResult = strictEqual(checkOne, checkTwo)
+    expect(checkResult).to.equal(true)
+  })
```

Comment:
Tests are always about the scenario, not the values. `strictEqual(5, 5)` and `strictEqual('str', 'str')` are the same test, they are both testing same type, same value. For something like this you'd want

1. Same type, same value (ex. 5, 5)
2. Same type, different value (ex. 5, 6)
3. Different type, same value (ex. 5, '5')
4. Different type, different value (ex. 5, 'a')